cmd/veneur/main.go

```diff
@@ -71,6 +71,14 @@ func main() {
 		}
 	}()
 
+	// Accept connections on the TCP port if enabled
+	go func() {
+		defer func() {
+			server.ConsumePanic(recover())
+		}()
+		server.AcceptTCP(server.shutdownChan)
```

Comment:
As is done with the trace reader above, can we limit this to when the config says it should run?